.gitignore

```diff
@@ -88,6 +88,7 @@ RNTester/build
 /template/ios/Pods/
 /template/ios/Podfile.lock
 /RNTester/Gemfile.lock
+/RNTester/vendor
```

Comment:
The `vendor` directory came for running `bundle install --path vendor`, which I did to avoid installing the gems as system gems. I can remove it from this PR. It would be nice to have some better instructions on how to set up the development environment for newbies.